cumulus/polkadot-parachain/src/command.rs

```diff
@@ -164,6 +170,32 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 				&include_bytes!("../chain-specs/asset-hub-kusama.json")[..],
 			)?),
 
+		// -- Asset Hub Rococo
+		"asset-hub-rococo-dev" =>
+			Box::new(chain_spec::asset_hubs::asset_hub_rococo_development_config()),
+		"asset-hub-rococo-local" =>
+			Box::new(chain_spec::asset_hubs::asset_hub_rococo_local_config()),
+		// the chain spec as used for generating the upgrade genesis values
+		"asset-hub-rococo-genesis" => Box::new(chain_spec::asset_hubs::asset_hub_rococo_config()),
+		// the shell-based chain spec as used for syncing
```

Comment:
Isn't this a new chain? Why the need to reference a shell?

Reply:
> Isn't this a new chain?

When this is merged, we want to redeploy actual live AssetHubRococo on Roocco (formerly known as "Rockmine") here: https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frococo-asset-hub-rpc.polkadot.io#/explorer, there is still some old version of `asset-hub-kusama-runtime`.

Should we announce it somewhere or can we just reonboard from the scratch (with new spec and/or existing validators/collators keys)?
@joepetrowski  What do you suggest?


> Why the need to reference a shell?

I am not sure what do you mean by this comment, can you, please, elaborate more?
I've just copied it from AssetHubKusama or AssetHubWestend and chain spec:
```
                // -- Asset Hub Kusama
		"asset-hub-kusama-dev" | "statemine-dev" =>
			Box::new(chain_spec::asset_hubs::asset_hub_kusama_development_config()),
		"asset-hub-kusama-local" | "statemine-local" =>
			Box::new(chain_spec::asset_hubs::asset_hub_kusama_local_config()),
		// the chain spec as used for generating the upgrade genesis values
		"asset-hub-kusama-genesis" | "statemine-genesis" =>
			Box::new(chain_spec::asset_hubs::asset_hub_kusama_config()),
		// the shell-based chain spec as used for syncing
		"asset-hub-kusama" | "statemine" =>
			Box::new(chain_spec::asset_hubs::AssetHubKusamaChainSpec::from_json_bytes(
				&include_bytes!("../chain-specs/asset-hub-kusama.json")[..],
			)?),
```

Reply:
@joepetrowski actually, I dont know how was `Rockmine` deployed, if directly with `asset-hub-kusama-runtime` or `shell-runtime`.

@BulatSaif dont you know? And also where is the actual genesis chain spec for: https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frococo-asset-hub-rpc.polkadot.io#/explorer ?

Reply:
There are some things that AH-Rococo has been used for by the ecosystem. I know @metricaez just did some Trappist stuff there. I think nuking it will cause problems, because it will mess up balances that have been teleported.

Ideally this is a runtime that can be upgraded to. It will probably break a lot of wallets/explorers who rely on `spec_name`. It's a pain but easier to deal with than unbalanced checking/reserve accounts.

@al3mart might have an opinion here too, knowing more of the ecosystem's use.

Reply:
I think and hope, it should work just to upgrade runtime,
If `shell-runtime` -> `asset-hub-kusama-runtime` works then also `asset-hub-kusama-runtime` -> `asset-hub-rococo-runtime` should work, right?

The storage should be ok?
Maybe just to check if any migrations should happen? Anything else?

Actual deployed version:
```
{
  specName: statemine
  implName: statemine
  authoringVersion: 1
  specVersion: 10,005
  implVersion: 0
  apis: [...]
  transactionVersion: 13
  stateVersion: 0
}
```

But in the `polkadot-sdk` for `AssetHubKusama` I see:
```
{
        spec_name: create_runtime_str!("statemine"),
	impl_name: create_runtime_str!("statemine"),
	authoring_version: 1,
	spec_version: 10000,
	impl_version: 0,
	apis: RUNTIME_API_VERSIONS,
	transaction_version: 13,
	state_version: 1,
}
```

---


@joepetrowski 
So if we want to do just runtime upgrade, we should probably change `RuntimeVersion` also to be like this:
```
        spec_name: create_runtime_str!("statemine"),
	impl_name: create_runtime_str!("statemine"),
	spec_version: 10006,
```
right?


@BulatSaif Do we know what `10,005` version is? When was it deployed? From which branch or wasm version?

Reply:
You technically _can_ break the `spec_name` in an upgrade, I think it just causes problems for node operators and block explorers etc.

Reply:
> You technically _can_ break the `spec_name` in an upgrade, I think it just causes problems for node operators and block explorers etc.

ok, so lets go with `"statemine"`, we can do this cleanup from `statemine` -> `asset-hub-rococo` later if needed or when we will reonboard asset-hub-rococo from the scrach

Reply:
@BulatSaif ok, cool, thank you, it helped a lot,
so I checked that cumulus branch `remotes/origin/rockmint-10005` which is almost the same as master, and there is just deps update and runtime version changed to 10005,
so in this PR a I changed runtime version for AssetHubRococo 10006, so everything should be ok